cmd/get/profiles/profiles.go

```diff
@@ -0,0 +1,132 @@
+package profiles
+
+import (
+	"encoding/json"
+	"errors"
+	"fmt"
+	"net/http"
+
+	"github.com/MakeNowJust/heredoc"
+	"github.com/loginradius/lr-cli/api"
+	"github.com/loginradius/lr-cli/cmdutil"
+	"github.com/loginradius/lr-cli/config"
+	"github.com/loginradius/lr-cli/request"
+
+	"github.com/spf13/cobra"
+)
+
+var inpEmail string
+var inputUID string
+
+type Result struct {
+	FirstName string `json:"FirstName"`
+	Uid       string `json:"Uid"`
+	ID        string `json:"ID"`
+}
+
+func NewprofilesCmd() *cobra.Command {
+
+	cmd := &cobra.Command{
+		Use:     "profiles",
+		Short:   "get profiles",
+		Long:    `This commmand gets profiles`,
+		Example: heredoc.Doc(`$ lr get profiles --email <email> (or) --UID <UID>`),
+		RunE: func(cmd *cobra.Command, args []string) error {
+			if inpEmail == "" && inputUID != "" {
+				return addbyUID(inputUID)
+			}
+			if inputUID == "" && inpEmail != "" {
+				return addbyEmail(inpEmail)
+			}
+			if inputUID == "" && inpEmail == "" {
+				return &cmdutil.FlagError{Err: errors.New("atleast one of the flags is necessary")}
+			}
+
+			return addbyEmail(inpEmail)
+		},
+	}
+
+	fl := cmd.Flags()
+	fl.StringVarP(&inpEmail, "email", "e", "", "emailID")
+	fl.StringVarP(&inputUID, "uid", "u", "", "UID")
+
+	return cmd
+}
+
+func addbyEmail(inpEmail string) error {
+	var resObj *cmdutil.APICred
+	res, err := cmdutil.GetAPICreds()
+	if err == nil {
+		resObj = &cmdutil.APICred{
+			Key:    res.Key,
+			Secret: res.Secret,
+		}
+	} else {
+		resp, err := api.GetSites()
+		if err != nil {
+			return err
+		}
+		resObj = &cmdutil.APICred{
+			Key:    resp.Key,
+			Secret: resp.Secret,
+		}
+		cmdutil.StoreAPICreds(resObj)
+	}
+
+	var emailstring = inpEmail
+	url := config.GetInstance().LoginRadiusAPIDomain + "/identity/v2/manage/account?apikey=" + resObj.Key + "&apisecret=" + resObj.Secret + "&email=" + emailstring
+	var resultResp Result
+	resp, err := request.Rest(http.MethodGet, url, nil, "")
+
+	if err != nil {
+		return err
+	}
+	err = json.Unmarshal(resp, &resultResp)
+	if err != nil {
+		return err
+	}
+	fmt.Println("First name is:" + resultResp.FirstName)
+	fmt.Println("Uid is:" + resultResp.Uid)
+	fmt.Println("ID is:" + resultResp.ID)
+
+	return nil
+}
+
+func addbyUID(inputUID string) error {
+	var resObj *cmdutil.APICred
+	res, err := cmdutil.GetAPICreds()
+	if err == nil {
+		resObj = &cmdutil.APICred{
+			Key:    res.Key,
+			Secret: res.Secret,
+		}
+	} else {
+		resp, err := api.GetSites()
+		if err != nil {
+			return err
+		}
+		resObj = &cmdutil.APICred{
+			Key:    resp.Key,
+			Secret: resp.Secret,
+		}
+		cmdutil.StoreAPICreds(resObj)
+	}
+
+	var uidstring = inputUID
+	url := config.GetInstance().LoginRadiusAPIDomain + "/identity/v2/manage/account/" + uidstring + "?apikey=" + resObj.Key + "&apisecret=" + resObj.Secret
```

Comment:
For profiles, there must be some other URL.
`/identity/v2/manage/account` - this endpoint fetches the accounts.